clouddriver/clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClient.groovy

```diff
@@ -499,7 +499,7 @@ class DockerRegistryClient {
     }
     if (queryParamsString) {
       queryParams = queryParamsString.split("&").collectEntries { param ->
-        def (key, value) = param.split("=")
+        def (key, value) = param.split("=", 2)
```

Comment:
I could use a comment here about why 2 is correct/important.

Reply:
Will add a comment. It means, it returns two items even when there is no param value.